templates/kwoc/end-eval-reminder.txt

```diff
@@ -0,0 +1,12 @@
+KWoC : End-Evaluation Reminder!!
+
+Dear Participant,<br><br>
+
+We hope this email finds you well. Your hard work and dedication throughout the program have been truly commendable, and we greatly appreciate the efforts you have put into the project<br><br>
+
+This is to remind you that the end-evaluation deadline for <b>Kharagpur Winter of Code<b> program is fast approaching. The deadline for the end-evaluation is {end_eval_date}.<br><br>
```

Comment:
I think we should mention that the participant should check their dashboard to complete the end evaluation.